wwwapp/models.py

```diff
@@ -60,6 +60,48 @@ def __str__(self):
         return '%s: %s, %s' % (self.year, self.user_profile, self.status)
 
 
+class PESELField(models.CharField):
+    """PESEL field, with checksum verification."""
+
+    def validate(self, pesel: str, user_info: 'UserInfo') -> None:
+        super().validate(pesel, user_info)
+        # We accept empty PESEL (don't raise exception) for legacy reasons.
+        if not pesel:
+            return
+
+        # https://en.wikipedia.org/wiki/PESEL#Format
+        if len(pesel) != 11:
+            raise ValidationError('PESEL length is wrong ({})'.format(len(pesel)))
```

Comment:
Shouldn't the user-visible errors be in Polish?

Reply:
We should change it to Polish for now, or use some i18n ;) (https://docs.djangoproject.com/en/2.2/topics/i18n/)